static/skin/index.css

```diff
@@ -105,7 +105,7 @@ body {
     border-radius: 10px;
     border: solid 1px #b5b2b2;
     padding: 10px;
-    background-image: url('./search-icon.svg');
+    background-image: url('../skin/search-icon.svg?KIWIXCACHEID');
```

Comment:
Why the change in the path (`../skin`) ?

Reply:
Because relative URLs to static resources are recognized by the `kiwix-resources` script using the `/skin/` anchor